lib/web/apiserver.go

```diff
@@ -3884,6 +3907,11 @@ func SSOSetWebSessionAndRedirectURL(w http.ResponseWriter, r *http.Request, resp
 // GET /webapi/sites/:site/auth/export?type=<auth type>
 // GET /webapi/auth/export?type=<auth type>
 func (h *Handler) authExportPublic(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
+	err := h.rateLimitRequest(r)
```

Comment:
Why did you decide to implement manual limiting here inside the handler as opposed to using a middleware like you did for all the other endpoints?